bin/fuel-core/src/cli/run/p2p.rs

```diff
@@ -170,12 +170,15 @@ pub struct P2PArgs {
 
 #[derive(Debug, Clone, Args)]
 pub struct SyncArgs {
```

Comment:
I'm not sure I understand. Which piece of code is looking for these values?

Reply:
I meant that while receiving the `SealedHeaders` on the receiver side of the P2P, we need to limit the size of the batch. Because right now, anyone can request 1B headers without punishment.

Reply:
I think I understand. What is the correct behavior of the sender? Should they truncate the request or should they error?

I think I'll just truncate for now. Seems fine.

Reply:
Based on the current `fuel-core-sync` implementation, it is better to reject the request=D Because if we request 100 batches and the first batch fails, we will waste a lot of resources for nothing(because all batches, except the first, will be dropped).

Reply:
Ah. Kk. Fair enough.